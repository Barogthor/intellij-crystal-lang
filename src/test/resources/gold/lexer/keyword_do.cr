get "/" do |ctx|
  ctx.response.content_type = "text/plain"
  "Hello, Auth0!"
end