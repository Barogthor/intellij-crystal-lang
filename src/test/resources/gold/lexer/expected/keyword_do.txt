IDENT ('get')
WHITE_SPACE (' ')
BAD_CHARACTER ('"')
/ ('/')
BAD_CHARACTER ('"')
WHITE_SPACE (' ')
do ('do')
WHITE_SPACE (' ')
| ('|')
IDENT ('ctx')
| ('|')
WHITE_SPACE ('\n  ')
IDENT ('ctx')
. ('.')
IDENT ('response')
. ('.')
IDENT ('content_type')
WHITE_SPACE (' ')
= ('=')
WHITE_SPACE (' ')
BAD_CHARACTER ('"')
IDENT ('text')
/ ('/')
IDENT ('plain')
BAD_CHARACTER ('"')
WHITE_SPACE ('\n  ')
BAD_CHARACTER ('"')
STRING ('H')
IDENT ('ello')
, (',')
WHITE_SPACE (' ')
STRING ('A')
IDENT ('uth0')
! ('!')
BAD_CHARACTER ('"')
WHITE_SPACE ('\n')
end ('end')